index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+$db = new PDO(
+    'mysql:host=DB;dbname=collections',   //DSN
+    'root',                    // Database username
+    'password'                 //Database password
+);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `location`, `sublocation`, `attraction`,`info`, `photo`, `rating`  FROM `USA`");
+$result = $query->execute();
+
+$location=[];
+
+if($result) {
+    $all = $query->fetchall();
+}
+foreach ($all as $item) {
+    echo "The location is " . $item['location'];
+    echo "<br>";
+    echo "The sublocation is " . $item['sublocation'];
+    echo "<br>";
+    echo "While I was there I went to " . $item['attraction'];
+    echo "<br><br>";
+}
```

Comment:
Would be a good idea to move this into it's own function, one that you can pass in the array and will return all the HTML needed to display it. This can be unit tested as well